apps/rush-lib/src/api/test/RushConfiguration.test.ts

```diff
@@ -118,6 +118,86 @@ describe('RushConfiguration', () => {
     done();
   });
 
+  it('can load repo/rush-npm-extended-project.json', (done: jest.DoneCallback) => {
+    const rushFilename: string = path.resolve(__dirname, 'repo', 'rush-npm-extended-projects.json');
+
+    const rushConfiguration: RushConfiguration = RushConfiguration.loadFromConfigurationFile(rushFilename);
+
+    rushConfiguration.projectsExtended = [
+      path.resolve(__dirname, 'repo', 'extended-projects', 'extended-1.json'),
+      path.resolve(__dirname, 'repo', 'extended-projects', 'extended-2.json')
+    ];
+
+    expect(rushConfiguration.packageManager).toEqual('npm');
```

Comment:
Most of this is superfluous to the test case

Reply:
Agreed and changed